### DISCUSSION TITLE
MIDI equipment and GrandOrgue

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#874] [General]
Hi!

We've migrated the wiki from sourceforge and started updating it, you can visit it from the navbar [Wiki link](https://github.com/GrandOrgue/grandorgue/wiki). Now, we plan to re-work the [Broken Midi adapters](https://github.com/GrandOrgue/grandorgue/wiki/Broken-MIDI-Adapters) page to a more general "MIDI equipment for GO" page. For this we kindly ask that you make a post in this discussion thread about what MIDI equipment you have used with GO, what has worked, and perhaps even more important: what did not work as intended. Thanks!

**Comment by jason-barba** [comment]
I use M-AUDIO Uno MIDI to USB converter (https://m-audio.com/products/view/uno). Works well with GO on Ubuntu 20.04 LTS.
